d2l-table.html

```diff
@@ -22,261 +61,37 @@
-				background-color: var(--d2l-table-row-background-color-active-selected);
-			}
-
-			:host-context([dir="rtl"]) ::content > tbody > tr[active][selected] >:first-child,
-			:host-context([dir="rtl"]):host([selectable]) ::content > tbody > tr[selected]:hover >:first-child,
-			:host ::content > tbody > tr[active][selected] >:last-child,
-			:host([selectable]) ::content > tbody > tr[selected]:hover >:last-child {
-				border-right-color: var(--d2l-table-row-border-color-active-selected) !important;
-				border-left-color: var(--d2l-table-border-color);
-			}
-
-			:host-context([dir="rtl"]) ::content > tbody > tr[active][selected] >:last-child,
-			:host-context([dir="rtl"]):host([selectable]) ::content > tbody > tr[selected]:hover >:last-child,
-			:host ::content > tbody > tr[active][selected] >:first-child,
-			:host([selectable]) ::content > tbody > tr[selected]:hover >:first-child {
-				border-left-color: var(--d2l-table-row-border-color-active-selected) !important;
-				border-right-color: var(--d2l-table-border-color);
-			}
-
-			:host ::content > tbody > tr[active][selected] td,
-			:host ::content > tbody > tr[active][selected] th,
-			:host([selectable]) ::content > tbody > tr[selected]:hover td,
-			:host([selectable]) ::content > tbody > tr[selected]:hover th,
-			:host ::content > tbody > tr[active][selected] + tr > td,
-			:host ::content > tbody > tr[active][selected] + tr > th,
-			:host([selectable]) ::content > tbody > tr[selected]:hover + tr > td,
-			:host([selectable]) ::content > tbody > tr[selected]:hover + tr > th {
-				border-top-color: var(--d2l-table-row-border-color-active-selected) !important;
-			}
-
-			:host ::content > tbody > tr:last-child[selected] td,
-			:host ::content > tbody > tr:last-child[selected] th {
-				border-bottom-color: var(--d2l-table-row-border-color-selected) !important;
-			}
-
-			:host ::content > tbody > tr:last-child[active][selected] td,
-			:host ::content > tbody > tr:last-child[active][selected] th,
-			:host([selectable]) ::content > tbody > tr:last-child[selected]:hover td,
-			:host([selectable]) ::content > tbody > tr:last-child[selected]:hover th {
-				border-bottom-color: var(--d2l-table-row-border-color-active-selected) !important;
-			}
-
-			:host ::content .d2l-table-col-sort-asc::after,
-			:host ::content .d2l-table-col-sort-desc::after {
-				display: inline-block;
-				vertical-align: middle;
-			}
-
-			:host ::content .d2l-table-col-sort-asc::after {
-				content: url('../d2l-icons/images/tier1/arrow-toggle-up.svg');
-			}
-
-			:host ::content .d2l-table-col-sort-desc::after {
-				content: url('../d2l-icons/images/tier1/arrow-toggle-down.svg');
 			}
 		</style>
 		<content></content>
 	</template>
 	<script>
 		Polymer({
 			is: 'd2l-table',
-			extends: 'table'
+			extends: 'table',
+			wrapMe: function() {
+				if (!this.parentNode) {
+					return;
+				}
+				if (this.parentNode.tagName === 'D2L-TABLE-WRAPPER') {
```

Comment:
If this ever gets used in a non-HTML5 document (e.g. XHTML, heaven forbid), I don't think the uppercase-ness is guaranteed. So you may want to explicitly `.toUpperCase()` it.